curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153908a366afb4d57185c8d.md

```diff
@@ -7,16 +7,23 @@ dashedName: step-12
 
 # --description--
 
-Flexbox can be thought of as having two axes, the main axis and the cross axis. The main axis is determined by the `flex-direction` property. If `flex-direction` is set to `row` or `row-reverse`, the main axis is horizontal. If `flex-direction` is set to `column` or `column-reverse`, the main axis is vertical.
+Flexbox has a main and cross axis. The main axis is defined by `flex-direction`, which has four possible values:
 
-Give your `#gallery` selector a `flex-direction` property set to `row`.
+- `row` (default): content direction, depending on writing mode.
+- `row-reverse`: opposite to the content direction.
+- `column`: top to bottom.
+- `column-reverse`: bottom to top.
+
+Try the different values to see how they affect the layout.
+
+When you are done, set an explicit `flex-direction` of `row` on the `.gallery` element.
```

Comment:
```suggestion
- `row` (default): horizontal axis with flex items from left to right
- `row-reverse`: horizontal axis with flex items from right to left
- `column`: vertical axis with flex items from top to bottom
- `column-reverse`: vertical axis with flex items from bottom to top

Try the different values to see how they affect the layout. When you are done, set the `flex-direction` of your `.gallery` to `row-reverse`.
```

Not sure if this is any better, wonder if we should use `row-reverse` here so it's not the default (test would need changing).

Reply:
The direction is writing mode agnostic. We can't say left to right. I originally used the CSS tricks list but it used an abbreviation for the writing mode so I combined it with some of MDNs text.

Reply:
That is a very long sentence to put after the values though. Maybe we/I can think of a way to incorporate the writing mode information in the main text before the values list.

Also, I don't think you meant to change the direction to `row-reverse` as that isn't what the rest of the code is like.

Reply:
Yea, the row-reverse was more of an idea than suggestion. I think it's a good idea since it's something other that the default - but we can leave it as the default for now.

Reply:
I just think doing it for no reason other than it not being the default is a little odd. I did initially try to think of a way to use a non-default value but then I would have had to create more steps. One idea was to start it out in a column before the wrapping had been applied and then switch it back to a row after.

The reverse properties are usually also discouraged from an accessibility standpoint as they visually change the order of the elements without it being reflected in the DOM.